### DISCUSSION TITLE
[Feature] Keep the selection after doing downloading

### DISCUSSION THREAD

**Discussion by Biepa** [immich-app/immich#12607] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I wanted to download a specific set of photos and then archive/delete them.
Unfortunately after downloading them the whole selection was deselected and the process has to be done again.
I think it would be a nice thing to keep the selection after the download action is performed (makes so nense for delete/archive) of course.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
(effectively) a duplicate of #2686

**Reply by Biepa** [reply]
That's right. I missed that.
Feel free to close this one.
